README.md

```diff
@@ -0,0 +1,37 @@
+# An action to start Filebrowser
+
+This action launches an instance of [Filebrowser](https://github.com/filebrowser/filebrowser) and opens it in the default browser.
+
+Has no required arguments.
+
+### Quick example
+
+```
+jobs:
+  browser:
+    action: gh:neuro-actions/registry_browser@master
```

Comment:
Recommended usage (in all readme's for all actions) is to use `@v1.0.0` instead of `@master`. We only use `@master` in test action in the repo, but all users should adhere to a specific fixed version (at the moment all actions have only the `v1.0.0`)

Reply:
Fixed in latest commit